static/js/stream_list.js

```diff
@@ -400,14 +432,14 @@ function rebuild_recent_private_messages(active_conversation) {
 
 exports.update_streams_sidebar = function () {
     exports.build_stream_list();
-
     if (! narrow.active()) {
         return;
     }
 
     var op_stream = narrow.filter().operands('stream');
     var op_subject = narrow.filter().operands('topic');
     var subject;
+    var i;
```

Comment:
It doesn't look like `i` is being used. Seems like you didn't intend for this to be included in the diff